tracer/build/_build/Build.Gitlab.cs

```diff
@@ -19,6 +22,97 @@
 
 partial class Build
 {
+    Target DownloadWinSsiTelemetryForwarder => _ => _
+       .Description("Downloads the telemetry forwarder executable used by SSI ")
+       .Unlisted()
+       .Requires(() => IsWin)
+       .Before(SignDlls)
+       .Executes(async () =>
+        {
+            // Download the forwarder from Azure for now.
+            // We will likely change this in the future, but it'll do for now.
+            const string url = "https://apmdotnetci.blob.core.windows.net/apm-datadog-win-ssi-telemetry-forwarder/telemetry_forwarder.exe";
+            const string expectedHash = "3A207302683E2BE8CB3D3A6AFF4F40721B7C85763A5A9B8F441D6BFD1B419F02D3EA8155634CEB3A09473F4745CE49679A55095F0EAD91F0582C6CB08C725782";
+
+            var tempFile = await TryDownloadForwarder();
+            var actualHash = GetSha512Hash(tempFile);
+            if (!string.Equals(expectedHash, actualHash, StringComparison.Ordinal))
+            {
+                throw new Exception($"Downloaded file did not have expected hash. Expected hash {expectedHash}, actual hash {actualHash}");
+            }
+
+            Logger.Information("Hash verified: '{Hash}'", expectedHash);
+
+            // Move to expected location
+            var output = ArtifactsDirectory / "telemetry_forwarder.exe";
+            FileSystemTasks.CopyFile(tempFile, output, FileExistsPolicy.Overwrite);
+
+            static async Task<string> TryDownloadForwarder()
```

Comment:
Both of these seem like good utility methods to have in general, could we place them in a common location?

Reply:
Moved them to `Build.Utilities` in https://github.com/DataDog/dd-trace-dotnet/pull/7342/commits/a1891190e6d2df17c3866803c225f16257254999 we could move them somewhere else even more general, not fussed 🤷‍♂️